Assets/Scripts/PowerUps/CameraShake.cs

```diff
@@ -0,0 +1,18 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class CameraShake : MonoBehaviour
+{
+    private Animator animator;
+
+    private void Start()
+    {
+        animator = GetComponent<Animator>();
+    }
+
+    public void Shake()
+    {
+        animator.SetTrigger("shake");
+    }
```

Comment:
Using Animations for shaking Camera might not be the most optimal thing, we should be manipulating camera's position wrt game screen.